pointerevents/pointerevent_attributes.html

```diff
@@ -0,0 +1,301 @@
+<!doctype html>
+<html>
+<head>
+  <title>Pointer Events properties tests</title>
+  <meta name="viewport" content="width=device-width">
+  <meta name="variant" content="?mouse">
+  <meta name="variant" content="?pen">
+  <meta name="variant" content="?mouse-right">
+  <meta name="variant" content="?pen-right">
+  <meta name="variant" content="?touch">
+  <meta name="variant" content="?mouse-nonstandard">
```

Comment:
"nonstandard" is maybe bit harsh—though I guess it is true because they're testing a non-normative statement ("Note: Many user agents expose non-standard attributes fromElement and toElement in MouseEvents to support legacy content. We encourage those user agents to set the values of those (inherited) attributes in PointerEvents to null to transition authors to the use of standardized alternates (i.e. target and relatedTarget).")

Reply:
It literally uses the term "non-standard" to describe those attributes in the note you're quoting :p